src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -4126,12 +4010,6 @@ resource pipeline_ToDataExplorer 'Microsoft.DataFactory/factories/pipelines@2018
         description: 'Read the hub config to determine how long data should be retained.'
         type: 'Lookup'
         dependsOn: [
-          {
-            activity: 'Wait'
-            dependencyConditions: [
-              'Succeeded'
-            ]
-          }
         ]
         policy: {
           timeout: '0.12:00:00'
```

Comment:
On your line 4073, you will need to remove the `Loop thru Columns` dependency. I missed that in a previous iteration 😔

Reply:
FYI - I resolved your merge conflict and fixed this